### PR TITLE
CSI-1752 add 1.3.0 CSV yamls

### PR DESCRIPTION
**Pull request by oriyarde**

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,766 @@
...
+    Supported IBM storage systems:
+      - IBM FlashSystem 9100
+      - IBM Spectrum Virtualize Family (including IBM Flash family members built with IBM Spectrum Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R) and IBM SAN Volume Controller (SVC) models SV2, SA2)
+      - IBM FlashSystem A9000/R
+      - IBM DS8880
+      - IBM DS8900
+
+    Supported operating systems:
+      - RHEL 7.x (x86 architecture)
+      - RHCOS (x86 and IBM Z architecture)
```
Comment (barakda) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml:131]: what about power arch?

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,766 @@
...
+    ```bash
+    $ kubectl -n <namespace> apply -f ibm-block-csi-operator.yaml
+    ```
+
+    ### Verify the operator is running:
+
+    ```bash
+    $ kubectl get pod -l app.kubernetes.io/name=ibm-block-csi-operator -n <namespace>
+    NAME                                    READY   STATUS    RESTARTS   AGE
+    ibm-block-csi-operator-5bb7996b86-xntss 2/2     Running   0          10m
```
Comment (barakda) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml:291]: I think there is only one container in operator pod (not 2)

File: deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,766 @@
...
+    Supported IBM storage systems:
+      - IBM FlashSystem 9100
+      - IBM Spectrum Virtualize Family (including IBM Flash family members built with IBM Spectrum Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R) and IBM SAN Volume Controller (SVC) models SV2, SA2)
+      - IBM FlashSystem A9000/R
+      - IBM DS8880
+      - IBM DS8900
+
+    Supported operating systems:
+      - RHEL 7.x (x86 architecture)
+      - RHCOS (x86 and IBM Z architecture)
```
Comment (barakda) [inline deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml:131]: same here - power arch is missing

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml (hunk 1 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml]: done

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml (hunk 2 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml]: done

File: deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml (hunk 3 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml]: done

**Review by ArbelNathan — APPROVED** [review]
LGTM

File: deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,766 @@
...
+    ```bash
+    $ kubectl -n <namespace> apply -f ibm-block-csi-operator.yaml
+    ```
+
+    ### Verify the operator is running:
+
+    ```bash
+    $ kubectl get pod -l app.kubernetes.io/name=ibm-block-csi-operator -n <namespace>
+    NAME                                    READY   STATUS    RESTARTS   AGE
+    ibm-block-csi-operator-5bb7996b86-xntss 2/2     Running   0          10m
```
[the remark this answers is not in the document]
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml]: already [done](https://github.com/IBM/ibm-block-csi-operator/pull/93/commits/1446c38066d731aa6de0831141c12750da6cf60a#diff-fb9d50ddde1b2ebebd8daafb746cda36)

**Pull request merged**